src/Math/RealArithmetic.dfy

```diff
@@ -20,6 +20,10 @@ module RealArith {
     Abs(x - y)
   }
 
+  function Pow(base: real, exponent: nat): real {
```

Comment:
Should we introduce a file `NatArithmetic.dfy` and move https://github.com/dafny-lang/Dafny-VMC/blob/37093b68240dd3d0592d799fe1f13500f795164a/src/Math/Helper.dfy#L11 from `Helper.dfy` into that new file? I am not entirely happy with the current distinction.

Reply:
`NatArith.dfy` already exists and has a todo for exactly that. I'll do it in a separate PR: https://github.com/dafny-lang/Dafny-VMC/blob/37093b68240dd3d0592d799fe1f13500f795164a/src/Math/NatArith.dfy#L10

Reply:
Oh, right, my fault. Appreciate a follow up PR.